board/embeddedartists/mx7ulpea-ucom/mx7ulpea-ucom.c

```diff
@@ -87,6 +126,250 @@ int board_early_init_f(void)
 	return 0;
 }
 
+/* stub TEE operations until ready to integrate with it*/
+static struct tee_hash t_hash;
+
+static int optee_get_tee_hash(uint8_t *tee_hash, int *tee_hash_len, int *payload_len)
```

Comment:
Should we conditionally enable the optee check by making sure CONFIG_OPTEE is enabled?